addons/payment_authorize/models/authorize_request.py

```diff
@@ -58,9 +58,17 @@ def _make_request(self, operation, data=None):
 
         messages = response.get('messages')
         if messages and messages.get('resultCode') == 'Error':
+            err_msg = messages.get('message')[0].get('text', '')
+
+            # TODO transactionResponse error handling should happen in auth_and_capture() and authorize()
```

Comment:
![do-or-do-not-there-is-no-todo](https://user-images.githubusercontent.com/15896813/147938469-4e3a4fa3-af67-43f8-b23c-8321ff91dc46.jpeg)

I agree that this could be improved, though. I overlooked this file a bit when refactoring payment modules, and this one definitely needs a bit of cleanup. I'm ok to leave the error handling in here until we clean the whole file at once.

Reply:
Ok, I removed the TODO.